packages/nextjs/utils/scaffold-eth/getMetadata.ts

```diff
@@ -30,5 +30,12 @@ export const getMetadata = ({
       description: description,
       images: [imageUrl],
     },
+    other: {
+      "fc:frame": "vNext",
+      "fc:frame:image": imageUrl,
+      "fc:frame:button:1": "Open my 🏗️ Scaffold-ETH 2 dApp",
```

Comment:
Should we use the **title** var here or add another element to the hash param?

Reply:
yep, maybe even something like `Go to ${title}`. {title} should be fine tho. What do you think?

Reply:
> maybe even something like Go to ${title}

Umm this will be misleading right ? If people share https://frames-se2.vercel.app/debug and if they have this as title : 

https://github.com/scaffold-eth/scaffold-eth-2/blob/699b259fe176f720bd9da9946ff7a02b14bd8f36/packages/nextjs/app/debug/page.tsx#L6

The btn would say `Go to Debug Contracts` but instead the button will take you to homepage 

😂 sorry for thinking too much on this lol its just abi.ninja routes coming to mind and lol 100% agree its not that  too common people sharing random routes of their website instead they mostly share home page 

But I think would prefer "Open my 🏗️ Scaffold-ETH 2 dApp" over `Go to ${title}`, people can always tweak it if they want

Reply:
lol yeah, I was thinking about this while posting the other comment 1 min ago.

Option (b) will fix it, and we won't need to have a dynamic title.

I personally enjoy these discussions, makes me really think what is best & simpler for what we are looking for.